packages/plugin-legacy/index.js

```diff
@@ -559,6 +565,44 @@ function replaceLegacyEnvBabelPlugin() {
   })
 }
 
+/**
+ * @param {any} targets
+ * @param {boolean} ignoreBrowserslistConfig
+ * @returns {'defaults'|undefined}
+ */
+function getDefaultTargets(targets, ignoreBrowserslistConfig) {
+  if (targets) return targets
+
+  if (ignoreBrowserslistConfig) {
+    return defaultTargets
```

Comment:
**nitpick (non-blocking):** You should stick to one formatting style ^^
So you could use a one-liner here `if (ignoreBrowserslistConfig) return defaultTargets` or use curly-braces on line 574

Applies also to line 592 and 598